Content.Server/Administration/Logs/AdminLogsEui.cs

```diff
@@ -103,41 +103,41 @@ public override async void HandleMessage(EuiMessageBase msg)
         switch (msg)
         {
             case LogsRequest request:
-            {
-                _sawmill.Info($"Admin log request from admin with id {Player.UserId.UserId} and name {Player.Name}");
-
-                _logSendCancellation.Cancel();
-                _logSendCancellation = new CancellationTokenSource();
-                _filter = new LogFilter
                 {
-                    CancellationToken = _logSendCancellation.Token,
-                    Round = request.RoundId,
-                    Search = request.Search,
-                    Types = request.Types,
-                    Impacts = request.Impacts,
-                    Before = request.Before,
-                    After = request.After,
-                    IncludePlayers = request.IncludePlayers,
-                    AnyPlayers = request.AnyPlayers,
-                    AllPlayers = request.AllPlayers,
-                    IncludeNonPlayers = request.IncludeNonPlayers,
-                    LastLogId = null,
-                    Limit = _clientBatchSize
-                };
-
-                var roundId = _filter.Round ??= CurrentRoundId;
-                await LoadFromDb(roundId);
-
-                SendLogs(true);
-                break;
-            }
+                    _sawmill.Info($"Admin log request from admin with id {Player.UserId.UserId} and name {Player.Name}");
```

Comment:
Why did you modify this EUI?

Reply:
Oops. What happened is I was looking at it since I was having trouble with Sawmill, and I accidentally pressed Ctrl+S, and then my auto-formatter kicked in and changed the spacing. Very dumb mistake I know lol